rest/admin_api.go

```diff
@@ -230,7 +230,11 @@ func (h *handler) handleGetDbConfig() error {
 			return base.HTTPErrorf(http.StatusNotFound, "database config not found")
 		}
 
-		h.response.Header().Set("ETag", dbConfig.Version)
+		quoteETag, err := h.setEtag(dbConfig.Version)
+		if err != nil {
+			return err
+		}
+		h.response.Header().Set("ETag", quoteETag)
```

Comment:
I expected `setEtag` to set the response header, not return the value for each handler to be manually setting it, given they're methods on a handler.